### ISSUE TITLE
Use next/image with focal_point module in Drupal

### ISSUE THREAD

**Issue by fiasco** [acquia/next-acms#71]
**Is your feature request related to a problem? Please describe.**
While we want FE developers to have greater autonomy over image display, we need to preserve features within the CMS such as the focal point module integration.

**Describe the solution you'd like**
As a FE developer, I would like to control image sizes with the next/image library but still allow end users to set focal points to crop around.

This probably means exposing large image styles of an image to the FE developer so they can size accordingly.

**Comment by robdecker** [comment]
We've used [Consumer Image Styles](https://www.drupal.org/project/consumer_image_styles) to do this. Using this module, you can then generate an image like so:

```jsx
import Image from 'next/image'

...

<Image
  src={paragraph.field_image?.field_media_image?.links['225x225']?.href}
  alt={paragraph.field_image?.field_media_image?.resourceIdObjMeta?.alt}
  width={paragraph.field_image?.field_media_image?.links['225x225']?.meta.width}
  height={paragraph.field_image?.field_media_image?.links['225x225']?.meta.height}
  className="rounded-full"
/>
```

This may not get you 100% of what you are looking for, but it's worked well for us.

**Comment by lauriii** [comment]
Thank you @robdecker! I was able to confirm that something along the lines of what you're proposing would indeed work. This means that coming up with a solution to https://github.com/acquia/next-acms/issues/37 should also address this.

**Comment by lauriii** [comment]
This has been resolved by #37.
